src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
```

Comment:
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
```
You can pass the default parameter as follow: